src/ConfigCatClient.php

```diff
@@ -155,10 +160,15 @@ public function getValue(string $key, mixed $defaultValue, ?User $user = null):
                     'event_id' => 1001,
                     'KEY' => $key,
                     'DEFAULT_PARAM_NAME' => '$defaultValue', 'DEFAULT_PARAM_VALUE' => Utils::getStringRepresentation($defaultValue),
-                    'AVAILABLE_KEYS' => implode(", ", array_keys($settingsResult->settings))
+                    'AVAILABLE_KEYS' => "'".implode("', '", array_keys($settingsResult->settings))."'"
```

Comment:
Although this would work when key count > 0 but it should result in an empty string otherwise.

Reply:
The case when `$settingsResult->settings` is empty is handled earlier and a different message is logged.

Reply:
Indeed, I missed that. However, this seems to open up another can of worms...

We have 2 cases regarding these evaluation precondition checks:
1. When we have no config JSON cached at client-side --> error #1000
2. When we have a config JSON cached but it doesn't contain the key --> error #1001

As far as I can see, we log error #1000 [in the wrong case](https://github.com/configcat/php-sdk/pull/31/files#diff-7118a23ef9dbdb128e4847cec4204b8bc59049835a6a22eb2dc19443a839cda3R138) (in case 2, i.e. when the cached config is available but empty) but [throw an exception at the location](https://github.com/configcat/php-sdk/blob/5aca2505233fcf28f4cac405b58eef6c45f9486a/src/ConfigCatClient.php#L636) where it should be actually logged.

Also, error #1001 should always be logged in case 2 regardless of whether the config is empty or not.